sklearn/ensemble/_gradient_boosting.pyx

```diff
@@ -8,8 +8,8 @@ import numpy as np
 from scipy.sparse import issparse
 
 from ..utils._typedefs cimport float32_t, float64_t, intp_t, int32_t, uint8_t
-# Note: _tree uses cimport numpy, cnp.import_array, so we need to include
-# numpy headers, see setup.py.
+# Note: _tree uses cimport numpy, cnp.import_array, so we don't need to do that
+# in this file
```

Comment:
the comment was to explain why numpy include dir was in the setup.py for this extension while it didn't seem to be necessary at first sight. 

It's still the case but now in meson.build so I'd just rephrase it
```suggestion
# Note: _tree uses cimport numpy, cnp.import_array, so we need to include
# numpy headers in the build configuration of this extension
```

Reply:
Right thanks, that makes a bit more sense now ...